djangocms_bootstrap4/contrib/bootstrap4_link/migrations/0001_initial.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2017-10-12 06:33
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import djangocms_attributes_field.fields
+import djangocms_link.validators
+
+from djangocms_link.models import get_templates, TARGET_CHOICES
+
+from ..constants import LINK_CHOICES, LINK_SIZES
+from ..models import COLOR_STYLES
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('cms', '0016_auto_20160608_1535'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Bootstrap4Link',
+            fields=[
+                ('template', models.CharField(choices=get_templates(), default=get_templates()[0][0], max_length=255, verbose_name='Template')),
+                ('name', models.CharField(blank=True, max_length=255, verbose_name='Display name')),
+                ('external_link', models.URLField(blank=True, help_text='Provide a valid URL to an external website.', max_length=2040, validators=[djangocms_link.validators.IntranetURLValidator(intranet_host_re=None)], verbose_name='External link')),
```

Comment:
set the intranet_host_validator to the setting, in order to avoid migration when it changes on project